doc-content/kogito-docs/src/main/asciidoc/creating-running/con_kogito-automation.adoc

```diff
@@ -53,4 +53,4 @@ Another focus of {PRODUCT} is optimum developer experience. You can use much or
 .Building a {PRODUCT} process service in VSCode
 image::kogito/kogito-run-quarkus-devmode-demo.gif[Demo of {PRODUCT} BPMN2 extension in VSCode]
 
-To deploy your services into the cloud, you can use the {PRODUCT} Operator, which guides you through every step. The {PRODUCT} Operator uses the https://github.com/operator-framework[Operator Framework] and automates many of the deployment steps for you. For example, you can provide the Operator a link to where your application code lives in Git, and the Operator can check it out, build it (including native compilation, if applicable), and deploy the resulting services. {PRODUCT} also offers a Command Line Interface (CLI) to simplify some of these deployment tasks.
+To deploy your services to the cloud, you can use the {PRODUCT} Operator, which guides you through the deployment process. The {PRODUCT} Operator uses the https://github.com/operator-framework[Operator Framework] and automates many of the deployment steps for you. For example, you can provide the Operator a link to where your application code lives in Git, and the Operator can check it out, build it (including native compilation, if applicable), and deploy the resulting services. {PRODUCT} also offers a Command Line Interface (CLI) to simplify some of these deployment tasks.
```

Comment:
In reply:
* How about "{PRODUCT} includes components **that are based on** well-known ..."? Just because to say Kogito as a whole is based on it makes it sound like there's little new in its own right. Wdyt?
* Changed to "... middleware technologies that {PRODUCT} **supports**"
* So like this? "You can expose your {PRODUCT} services with domain-specific APIs **that are** based on the processes and rules that you have defined **and that do not include third-party or internal APIs**."
* Sorry, mistake, missed this one. Changed all to this: "For example, you can provide the Operator a link to the Git repository that contains your application, and the operator can **pull the project**, build it (including native compilation, if applicable), and deploy the resulting services."

Reply:
Looks good to me! :)

Reply:
+1